src/features/deletePatient.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Delete Patient
+
+  Background:
+    Given setup logs in the system
+
+  @deletePatient
+  Scenario: Delete Patient
+  And  User clicks on search Patient Record app
```

Comment:
You can also start these steps with a `When User clicks on search Patient Record app` instead of starting with And